<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VTK_GUI_Qt</class>
 <widget class="QMainWindow" name="VTK_GUI_Qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleView</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidgetLeft" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>441</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Color with:</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PointData</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CellData</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Use array:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="updateButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Show/Hide Planes</string>
    </property>
    <widget class="QCheckBox" name="checkBoxX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxY">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxZ">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Positions</string>
    </property>
    <widget class="QSlider" name="verticalSliderX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>89</y>
       <width>22</width>
       <height>191</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSliderY">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>89</y>
       <width>22</width>
       <height>191</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSliderZ">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>89</y>
       <width>22</width>
       <height>191</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxX">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxY">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxZ">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxZ</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
